src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -87,6 +110,10 @@ export const TxParametersDetail = ({
     setIsAccordionExpanded(!isAccordionExpanded)
   }
 
+  const getColor = () => {
```

Comment:
Nice refactor! We could possibly move it outside the component or use `useMemo` since it returns the same result for all the places we use it.

Reply:
Do the parameters change at all?

Reply:
I don't think it actually needs to be a function, does it?